tsconfig.json

```diff
@@ -18,9 +18,9 @@
     "importHelpers": true,
     "strictNullChecks": true,
     "suppressImplicitAnyIndexErrors": true,
+    "ignoreDeprecations": "5.0",
     "noUnusedLocals": true
   },
-  "include": [
-    "**/*"
-  ]
+  "include": ["src/**/*"],
+  "exclude": ["src/**/*.spec.ts"]
```

Comment:
What's the goal in covering the unit test files with the integration test tsconfig?

Reply:
It's automatically removing decorator in integration test files when committing. Some husky command is behind that. I will remove these changes and commit it with --no-verify param.

Reply:
Unfortunately, that won't help as PRs automatically verify all of these changes before merging. Otherwise, the next person trying to contribute would run into the same error.
If you want, I can take some time tomorrow to clean up all of the setup to ensure that it just runs, and then you can focus on the actual feature - turns out I haven't worked with this project for a while and the dev dependency updates introduced some breaking changes.